### PR TITLE
Assign ALT to Command key on MacOS

### PR DESCRIPTION
**Pull request by tobiolo**
ALT is translated to Option key on MacOS
but shortcuts with Option + (Shift +) Number are used for characters on macOS. To not block input of characters, assign ALT+Number to CTRL+Number and thus to Command+Number on macOS.

### CODE REVIEW

File: src/mycanvas.h [hunk 1]
```diff
@@ -112,15 +112,16 @@ struct TSCanvas : public wxScrolledCanvas {
...
-        if ((ce.GetModifiers() == wxMOD_ALT || ce.GetModifiers() == (wxMOD_ALT | wxMOD_SHIFT)) &&
-            (ce.GetUnicodeKey() < 128)) {
-            ce.Skip();
-            return;
-        }
+        #ifndef __WXMAC__
+            // Without this check, Alt+[Alphanumericals], Alt+Shift+[Alphanumericals] and
+            // Alt+[Shift]+cursor (scrolling) don't work. The 128 makes sure unicode entry on e.g.
+            // Polish keyboards still works. (on Linux in particular).
+            if ((ce.GetModifiers() == wxMOD_ALT || ce.GetModifiers() == (wxMOD_ALT | wxMOD_SHIFT)) &&
```
Comment (tobiolo) [inline src/mycanvas.h:119]: On my Swiss german keyboard, I need to press Option+3 to generate # and this lies under 128. So just ignore this on wxMac.

**Comment by tobiolo** [comment]
Haha MSVC cannot deal macos anymore?

**Comment by tobiolo** [comment]
Now it works

File: src/myframe.h [hunk 2]
```diff
@@ -387,19 +387,79 @@ struct MyFrame : wxFrame {
                      _(L"Opens up the text from the selected cell in default application for the file type"));
 
             wxMenu *laymenu = new wxMenu();
-            MyAppend(laymenu, A_V_GS, _(L"Vertical Layout with Grid Style Rendering\tALT+1"));
-            MyAppend(laymenu, A_V_BS, _(L"Vertical Layout with Bubble Style Rendering\tALT+2"));
-            MyAppend(laymenu, A_V_LS, _(L"Vertical Layout with Line Style Rendering\tALT+3"));
+            MyAppend(laymenu, A_V_GS, 
+                #ifdef __WXMAC__
```
Comment (aardappel) [inline src/myframe.h:391]: #ifdef __WXMAC__
   #define CTRLORALT "ALT"
// etc
_(L"Vertical Layout with Grid Style Rendering\t" CTRLORALT  "+1")

Comment (tobiolo) [reply src/myframe.h:391]: Ah, thanks for the input! This is far better.
Is it good now?

**Comment by tobiolo** [comment]
Success! 😅 Thanks!

**Pull request merged**